gensim/models/_utils_any2vec.pyx

```diff
@@ -44,6 +50,31 @@ cpdef ft_hash(unicode string):
     return h
 
 
+cpdef ft_hash_bytes(bytes bytez):
```

Comment:
why not `ft_hash` (just replace previous impl)?

Reply:
Replaced